components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/ApplicationAuthenticationServiceTest.java

```diff
@@ -135,18 +122,10 @@ public void testGetFederatedAuthenticators() throws Exception {
     @Test
     public void testGetRequestPathAuthenticators() throws Exception {
 
-        mockFrameworkServiceComponent();
```

Comment:
whats the purpose of removing the mock framework component?

Reply:
For static mocking, the PowerMockTestCase/PowerMockIdentityBaseTest have to extends but it has limitation of using the initMocks() method. Therefore I remove static mocking which gives additional advantages of minimizing usage of mocking.